zephyr/src/indigo_api_callback_dut.c

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2020 Wi-Fi Alliance                                                */
+/* Copyright (c) 2023 Wi-Fi Alliance                                                */
```

Comment:
we cannot change original year here as we don't own this copyright. We could add below, 

`/* Copyright (c) 2023 Nordic semiconductor ASA */`